docs/Configuration-Settings.md

```diff
@@ -180,11 +180,26 @@ number.
     specified for `tachyon.master.hostname`.</td>
 </tr>
 <tr>
+  <td>tachyon.master.bind.host</td>
+  <td>0.0.0.0</td>
+  <td>The hostname Tachyon's master node binds to.</td>
+</tr>
+<tr>
   <td>tachyon.master.port</td>
   <td>19998</td>
   <td>The port Tachyon's master node runs on.</td>
 </tr>
 <tr>
+  <td>tachyon.master.web.hostname</td>
+  <td>localhost</td>
+  <td>The externally resolvable hostname of master's web service that a client uses to communicate with the service.</td>
+</tr>
+<tr>
+  <td>tachyon.master.web.bind.host</td>
+  <td>0.0.0.0</td>
+  <td>The hostname Tachyon's master's web server binds to.</td>
```

Comment:
Okay. We can use a fixed naming policy on the bind value. It can either 0.0.0.0 (default) or an externally resolvable hostname: 
1. If user did not specify the bind host, then the service is always bind to 0.0.0.0
2. If user specified the bind hostname, then user must use the same hostname on the connection. e.g.  In this case, the configuration of tachyon.master.web.hostname must be either tachyon.master.web.bind.host or not definded. 

Over all, the configuration of host names should follow this table:

| Specified Hostname |  Specified Bind Host   | Returned Connect Host |
|--------------------|------------------------|-----------------------|
| hostname           | hostname               | hostname              |
| not defined        | hostname               | hostname              |
| hostname           | 0.0.0.0 or not defined | hostname              |
| not defined        | 0.0.0.0 or not defined | localhost             |